spark/src/main/scala/com/alibaba/graphar/utils/IndexGenerator.scala

```diff
@@ -86,6 +86,19 @@ object IndexGenerator {
     spark.createDataFrame(rdd_with_index, schema_with_index)
   }
 
+  def generateVertexIndexColumnAndIndexMapping(vertexDf: DataFrame, primaryKey: String = ""): (DataFrame, DataFrame) = {
```

Comment:
NB: The method is added for generate index column and index mapping in one-shot, avoid to generate index mapping again when process edges.